.github/workflows/windows-2025-compatibility-test.yml

```
@@ -0,0 +1,755 @@
+            "Timestamp" = Get-Date -Format "yyyy-MM-dd HH:mm:ss UTC"
+          }
+          
+          New-Item -ItemType Directory -Path "${{ env.TEST_RESULTS_PATH }}" -Force | Out-Null
+          $perfResults | ConvertTo-Json | Out-File "${{ env.TEST_RESULTS_PATH }}\performance-${{ matrix.os }}-${{ matrix.config }}.json"
+          
+          Write-Output "score=$overallScore" >> $env:GITHUB_OUTPUT
+      
+      - name: Compatibility assessment
+        id: compatibility
+        shell: powershell
+        run: |
```

actionlint (1.7.7)

property "dependency-checks" is not defined in object type {performance: {conclusion: string; outcome: string; outputs: {string => string}}; system-info: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)